ecosystem/indexer-grpc/indexer-grpc-file-store/src/processor.rs

```diff
@@ -141,21 +152,69 @@ impl Processor {
             }
             // Drain the transactions buffer and upload to file store in size of multiple of BLOB_STORAGE_SIZE.
             let process_size = transactions_buffer.len() / BLOB_STORAGE_SIZE * BLOB_STORAGE_SIZE;
-            let current_batch = transactions_buffer.drain(..process_size).collect();
-
+            let current_batch: Vec<EncodedTransactionWithVersion> =
+                transactions_buffer.drain(..process_size).collect();
+            // Hack: Though the copy is expensive here but we want to inspect the transaction timestamp in the log without
+            // lifetime issue.
```

Comment:
Remove this hack comment? We can call out that this might be slow but that b/c it's the file store processor we don't have to be overly concerned with efficiency.